### PR TITLE
net-im/psimedia: readded/ported to gstreamer 1.x

### PR DESCRIPTION
**Pull request by Ri0n**
Closes: https://bugs.gentoo.org/650706

### CODE REVIEW

File: net-im/psimedia/psimedia-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils multilib eutils git-r3
```
Comment (a17r) [inline net-im/psimedia/psimedia-9999.ebuild]: Please use `cmake` instead. Neither `multilib` nor `eutils` are used in here.

File: net-im/psimedia/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+	<maintainer type="person">
+		<email>rion4ik@gmail.com</email>
+		<name>Sergey Ilinykh</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name='extras'>Enables extra non official patches</flag>
```
Comment (a17r) [inline net-im/psimedia/metadata.xml]: extra extra ;) - rather describe it similar to `net-im/psi` as `Enable unofficial Psi+ patches`?

Please use double quotes everywhere in this file.

File: net-im/psimedia/psimedia-9999.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	dev-qt/qtwidgets:5
+	media-libs/gstreamer:1.0
+	media-libs/gst-plugins-base:1.0
+	media-libs/gst-plugins-good:1.0
+"
+RDEPEND="${DEPEND}
+	media-plugins/gst-plugins-jpeg:1.0
+	media-plugins/gst-plugins-opus:1.0
+	media-plugins/gst-plugins-v4l2:1.0
+	>=net-im/psi-9999[extras?]
```
Comment (a17r) [inline net-im/psimedia/psimedia-9999.ebuild]: Maybe you want to use `${PV}` in here so if you create a release some time in the future this is an easy copy?

File: net-im/psimedia/psimedia-9999.ebuild (hunk 1 above)
Comment (Ri0n) [reply net-im/psimedia/psimedia-9999.ebuild]: done

File: net-im/psimedia/metadata.xml (hunk 2 above)
Comment (Ri0n) [reply net-im/psimedia/metadata.xml]: there are no patches now but rewritten

File: net-im/psimedia/psimedia-9999.ebuild (hunk 3 above)
Comment (Ri0n) [reply net-im/psimedia/psimedia-9999.ebuild]: thanks. updated.

File: net-im/psimedia/psimedia-9999.ebuild (hunk 1 above)
Comment (a17r) [reply net-im/psimedia/psimedia-9999.ebuild]: still seeing bogus `multilib` and `eutils` there ;)

File: net-im/psimedia/psimedia-9999.ebuild [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake multilib eutils git-r3
```
Comment (a17r) [inline net-im/psimedia/psimedia-9999.ebuild]: still seeing bogus `multilib` and `eutils` there ;)

Comment (Ri0n) [reply net-im/psimedia/psimedia-9999.ebuild]: oh missed it. Fixed now.

**Pull request closed without merging**